src/renderer/ducks/validator/sagas.ts

```diff
@@ -539,11 +556,25 @@ export function* watchValidatorDuties({
     function* processDuties(
         epoch: number,
     ): Generator<CallEffect | AllEffect<CallEffect>, void, AttesterDuty[] & ProposerDuty[]> {
-        const attestations = yield call(eth2API.validator.getAttesterDuties, epoch, [validatorState.index]);
-        const attestationsFuture = yield call(eth2API.validator.getAttesterDuties, epoch + 1, [validatorState.index]);
-        const propositions = yield call(eth2API.validator.getProposerDuties, epoch, [validatorId]);
-        const propositionsFuture = yield call(eth2API.validator.getProposerDuties, epoch + 1, [validatorId]);
-        const propositionsSuperFuture = yield call(eth2API.validator.getProposerDuties, epoch + 2, [validatorId]);
+        let attestations: AttesterDuty[], attestationsFuture: AttesterDuty[];
+        try {
+            attestations = yield retry(2, 0, eth2API.validator.getAttesterDuties, epoch, [validatorState.index]);
+            attestationsFuture = yield call(eth2API.validator.getAttesterDuties, epoch + 1, [validatorState.index]);
+        } catch {
```

Comment:
log errors

Reply:
this try-catch block is in the case of some sort of error, we don't want to break application for failing to fetch some meta data for nonessential work for validating

Reply:
not sking you to break app, just log error message instead of ignoring it so we have some info when debugging